source/features/reactions-avatars.tsx

```diff
@@ -1,5 +1,7 @@
 import './reactions-avatars.css';
 import React from 'dom-chef';
+import onetime from 'onetime';
+import {observe} from 'selector-observer';
 import select from 'select-dom';
 import {flatZip} from 'flat-zip';
 import * as pageDetect from 'github-url-detection';
```

Comment:
Sort

Reply:
Is there any guidelines for this other than looking at other files? I know some of them but it's not clear.